scheduler/src/cook/config.clj

```diff
@@ -393,7 +397,12 @@
                        ((util/lazy-load-var 'clojure.tools.nrepl.server/start-server) :port port)))
 
      :pools (fnk [[:config {pools nil}]]
-              pools)
+              (cond-> pools
+                (:job-resource-adjustment pools)
+                (update :job-resource-adjustment
```

Comment:
You should deconstruct the pools using fnk, instead of using this update approach.

Reply:
will refactor later